contracts/PetPark.sol

```diff
@@ -1,7 +1,82 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
+contract PetPark{
 
-contract PetPark {
+    // Initial animal types
```

Comment:
When using comments the general convention is to use natspec format - https://docs.soliditylang.org/en/v0.5.10/natspec-format.html